doc/whatsnew/2.13.rst

```diff
@@ -120,6 +120,10 @@ Extensions
 
 * ``DocStringStyleChecker``
 
+    * Re-enable checker ``bad-docstring-quotes`` for Python <= 3.7.
+
+      Closes #6087
```

Comment:
We could maybe remove this and the note about removing it below from this whatsnew given it's in the same minor version?

Reply:
Makes sense!

Reply:
I noticed we assigned this PR to the 2.14 milestone so I've moved this line to 2.14 instead 😄